src/index.ts

```diff
@@ -39,16 +52,68 @@ const plugin: JupyterFrontEndPlugin<void> = {
           return;
         }
         const urls = paths.map(path => decodeURIComponent(path));
-        app.restored.then(() => {
-          // use the JupyterLab command to open the file from the URL
-          urls.forEach(url => {
-            void commands.execute('filebrowser:open-url', { url });
```

Comment:
But it would be worth reconsidering some time in the future to unify how files are fetched and opened in both JupyterLab and the different Notebook 7 applications.

Reply:
Yeah, I think it's a good idea that we can maybe move faster here, and then upstream back?